pkg/cmd/kafka/update/update.go

```diff
@@ -281,20 +282,23 @@ func promptConfirmUpdate(opts *options) (bool, error) {
 // creates a summary of what values will be changed in this update
 // returns a formatted string. Example:
 // owner: foo_user	➡️	bar_user
+// to do: Needs update once reauthentication_enabled is incorporated
```

Comment:
How reauthentication_enabled affects this preview?

Reply:
The methods generating preview iterate through the entire update struct, the broken preview did display `reauthentication_enabled` too.

Reply:
Yes. That iteration thru all elements of struct is core issue IMHO

Reply:
We need to filter out the nil properties of the update object.